src/allmydata/crypto/__init__.py

```diff
@@ -0,0 +1,21 @@
+class BadSignature(Exception):
```

Comment:
These exceptions should go into a module rather than being part of the package `__init__.py`.  Also the package should have a docstring.